lib/tachikoma/application.rb

```diff
@@ -99,6 +100,13 @@ def bundler
             @parallel_option
           ].compact))
           sh(*%w(bundle update))
+
+          # restore_bundled_with
+          lock_file_contents = File.read(@bundler_lock_file)
```

Comment:
or `sh(['bundle', 'exec', 'restore-bundled-with', '--lockfile', @bundler_lock_file])`
Which is better?